spec/support/omniauth.rb

```diff
@@ -2,15 +2,10 @@
   config.before(:suite) do
     OmniAuth.configure do |omniauth_config|
       omniauth_config.test_mode = true
-      omniauth_config.mock_auth[:facebook] =
-        OmniAuth::AuthHash.new(
-          provider: 'facebook',
-          uid: SecureRandom.random_number(2**48),
-          info: {
-            name: FactoryGirl.generate(:name),
-            email: FactoryGirl.generate(:email)
-          }
-        )
     end
   end
+
+  config.before(:each) do
```

Comment:
Any way to restrict this to only for feature specs? calling like this for every model spec is wasteful